src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Abstractions/IAirship.cs

```diff
@@ -28,6 +28,15 @@ public DeepLinkEventArgs(string deepLink)
         }
     }
 
+    public class MessageCenterEventArgs : EventArgs
+    {
+        public string MessageId { get; internal set; }
```

Comment:
Do we need to specify that this is optional?

Reply:
Seems that setting a default value makes it optional, not sure if there's another way to do it?

Reply:
We can't mark strings as nullable, that would require Xamarin 8.0 language.

Reply:
@marc-scig I would still need to pass it as an argument in that case. Do we want it to be optional or nullable?

Reply:
Optional seems fine to me

Reply:
Yeah I'd probably go with optional here